server/src/main/java/org/server/SupServer.java

```diff
@@ -112,4 +131,78 @@ public void tellSomeone(String message, String fromname, String toname) {
             } catch (Exception ex) { ex.printStackTrace(); }
         }
     }
-}
\ No newline at end of file
+    
+    /**
+     * return the status to the specific one
+	 *
+	 * @param 
+	 * 	status - status info
+	 *  toname - the specific one to transfer status
+     * */
+    public void tellSomeoneStatus(String status, String toname)
+    {
+    	System.out.println("Return the \"" + status +"\" to " + toname);
+    	if (Contacts.getInstance().hasContact(toname)) {
+            try {
+                PrintWriter writer = Contacts.getInstance().getContact(toname);
+                writer.println(status);
```

Comment:
this was right, but we want to use common sending code now (after #4 gets merged). so it will just use "Utils.sendMessage()" instead